packages/@expo/cli/src/utils/codesigning.ts

```diff
@@ -35,14 +39,20 @@ export type CodeSigningInfo = {
    * An empty array indicates that there is no need to serve the certificate chain in the multipart response.
    */
   certificateChainForResponse: string[];
+  /**
+   * Scope key cached for the project when certificate is development Expo Go code signing.
+   * For project code signing (keyId == the projects generated keyId) this is undefined.
+   */
+  scopeKey: string | null;
 };
 
 type StoredDevelopmentExpoRootCodeSigningInfo = {
   easProjectId: string | null;
+  scopeKey: string | null;
   privateKey: string | null;
   certificateChain: string[] | null;
 };
-const DEVELOPMENT_CODE_SIGNING_SETTINGS_FILE_NAME = 'development-code-signing-settings.json';
+const DEVELOPMENT_CODE_SIGNING_SETTINGS_FILE_NAME = 'development-code-signing-settings-2.json';
```

Comment:
I believe we need to be careful about backwards compatibility since this file is shared across all instances of the CLI. While I _think_ old versions wouldn't have a problem loading the new format, it seems like it's probably not worth the complexity/maintenance tradeoff just to save a small amount of disk space.